swift/Sources/Blockchains/Cosmos.swift

```diff
@@ -0,0 +1,10 @@
+// Copyright © 2017-2019 Trust Wallet.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+import Foundation
+
+public typealias TWCosmosSigningInput = TW_Cosmos_Proto_SigningInput
+public typealias TWCosmosSendCoinsMessage = TW_Cosmos_Proto_SendCoinsMessage
```

Comment:
can you add `TW_Cosmos_Proto_Coin` and `TW_Cosmos_Proto_SigningOutput`

Reply:
> can you add `TW_Cosmos_Proto_Coin` and `TW_Cosmos_Proto_SigningOutput`

done in e3d847ffce5590b156c57274e78ee9b279977597